src/errors.js

```diff
@@ -163,6 +163,9 @@ exports.noExecutors = remediation =>
 exports.unknownExclude = excluded =>
     new exports.BadRequest('UNKNOWN_EXCLUDE', `Excluded Executor [${excluded}] not found in list of identified executors`);
 
+exports.unauthorizedGeneration = account =>
+    new exports.Forbidden(`Account ${account} is unauthorized to access this playbook`);
```

Comment:
This error will be confusing.  Its not the the account that does not have the right permission, but rather the certificate being presented. Perhaps change this to accept `req.identity.system.cn` instead and change the error message appropriately?

Reply:
Will do!